composer.json

```diff
@@ -24,9 +24,9 @@
   "license": "MIT",
   "require": {
     "php": ">=7.2",
-    "typo3/cms-core": "^10.0",
-    "friendsoftypo3/headless": "^2.0",
-    "gridelementsteam/gridelements": "^10.0"
+    "typo3/cms-core": "^11.5",
+    "friendsoftypo3/headless": "^3.0",
+    "gridelementsteam/gridelements": "*"
```

Comment:
I think it's better to require `11.0.0-dev`

see: https://gitlab.com/coderscare/gridelements/-/blob/master/composer.json#L5

Reply:
@schloram, I wouldn't pin both TYPO3 and EXT:gridelements version here, otherwise we'd have to adjust that constraint in a couple of weeks again when Jo releases the new version. Pinning the TYPO3 version should be sufficient IMO.

Reply:
@kitzberger Yes but in my opinion pinning a "real" version is mandatory instead of require "all versions of that package".

Imagine Jo drops a feature we are using in our extension with a future version. Then every current and new installation of headless_gridelements would break. This is why adjusting the version every time Jo releases a new version is the only (and safest) way to react to some breaking changes.